packages/backend/tests/handlers/remote-schemas/resolvers/daohaus/resolver.test.ts

```diff
@@ -0,0 +1,116 @@
+import nock from 'nock';
+
+import { Resolver } from '../../../../../src/handlers/remote-schemas/autogen/types';
+import { getDaoHausMemberships } from '../../../../../src/handlers/remote-schemas/resolvers/daohaus/resolver';
+
+type MockResolver<TResult, TParent, TContext, TArgs> = (
+  parent?: TParent | any,
```

Comment:
```suggestion
  parent?: TParent | unknown,
```

Reply:
I think using `unknown` instead of `any` for all of these will clear up the type errors in the pull request.

Reply:
So I took a look and there are a couple problems:
* linting happens differently on commit (no linting problems) and running from the root dir (brings up these linting errors)
* the root level lint hammers the entire set of code instead of obeying the configurations, which are configured to ignore the test directories
* `unknown` brings me to the same problem I had before: I need to do a bunch of type manipulation to make this work and that seems like a waste of time to me

I propose the fix is to do the following:
* leave the `any` (it is the [recommended solution](https://github.com/dotansimha/graphql-code-generator/issues/1763))
* add a `lint` command to each package that is missing it
* replace the root-level `lint` with `lerna run lint --parallel` so that it obeys the `.eslintrc` config

Reply:
> 
> I propose the fix is to do the following:
> 
>     * leave the `any` (it is the [recommended solution](https://github.com/dotansimha/graphql-code-generator/issues/1763))
> 
>     * add a `lint` command to each package that is missing it
> 
>     * replace the root-level `lint` with `lerna run lint --parallel` so that it obeys the `.eslintrc` config

Yup, agree with all of this.  Wrangling types is sometimes not worth it for test code. You should be able to add an eslint override for the `test` directory also (see the `.eslintrc` in backend project for an example)

Reply:
So I misremembered how eslint works and I thought the pathing overrode the config (I know it happens with some other package but clearly my mind is going). Either way, I'm in progress on this but wasn't able to wrap it up this morning. Will get back to it in a couple days when I have time again.